SPECS/python-termcolor/python-termcolor.spec

```
@@ -0,0 +1,140 @@
+%global pypi_name termcolor
+%global sum ANSI Color formatting for output in terminal
+Summary:        %{sum}
+Name:           python-%{pypi_name}
+Version:        1.1.0
+Release:        29%{?dist}
```

Looks like we forgot to increment this to match the changelog

```suggestion
Release:        30%{?dist}
```